cluster/terraform/main.tf

```diff
@@ -92,37 +102,32 @@ module "eks" {
   vpc_id     = module.vpc.vpc_id
   subnet_ids = module.vpc.private_subnets
 
-  create_cloudwatch_log_group              = false
-  create_cluster_security_group            = false
-  create_node_security_group               = false
-  authentication_mode                      = "API_AND_CONFIG_MAP"
-  enable_cluster_creator_admin_permissions = true
+  create_cloudwatch_log_group = false
```

Comment:
might want to add a note in the future about why this is (i.e. - so that you don't face issues when creating/destroying clusters of the same name repeatedly in succession). in theory, most will want to control this through their IaC